modules/osd-create-cluster-ccs.adoc

```diff
@@ -151,9 +151,23 @@ To successfully create a cluster, you must select *Enable Secure Boot support fo
 +
 endif::osd-on-gcp[]
 .. Leave *Enable user workload monitoring* selected to monitor your own projects in isolation from Red Hat Site Reliability Engineer (SRE) platform metrics. This option is enabled by default.
+.. Optional: Expand *Advanced Encryption* to make changes to encryption settings.
+ifdef::osd-on-aws[]
+.. Optional: Accept the default setting *Use default KMS Keys* to use your default AWS KMS key, or select *Use Custom KMS keys* to use a custom KMS key.
+... Optional: Enter the AWS Key Management Service (KMS) custom key Amazon Resource Name (ARN) ARN in the *Key ARN* field.
+// ifdef::osd-on-gcp[]
+// encryption keys through the Google Cloud Key Management Service.
+// endif::osd-on-gcp[]
+The key is used for encrypting all control plane, infrastructure, worker node root volumes, and persistent volumes in your cluster.
+//Commented out due to changes in the UI
```

Comment:
Up to you but if this is a permanent change you could just remove this rather than commenting it out. Sometimes in large files the // stuff gets lost in the ifdef conditionals.